targets/psa_switch/externs/psa_checksum.h

```diff
@@ -0,0 +1,41 @@
+
+
+#ifndef PSA_SWITCH_PSA_CHECKSUM_H_
+#define PSA_SWITCH_PSA_CHECKSUM_H_
+
+#include <bm/bm_sim/extern.h>
+#include <bm/bm_sim/calculations.h>
+
+
+namespace bm {
+
+namespace psa {
+
+class PSA_Checksum : public bm::ExternType {
+ public:
+  static constexpr p4object_id_t spec_id = 0xfffffffd;
```

Comment:
unused

Reply:
Yes. I will delete it. I thought I would use it but I didn't.